shared/src/main/tut/README.md

```diff
@@ -709,6 +709,123 @@ The `UnitGroup` values provided with Squants are only samples and aren't intende
 We encourage users to make their own `UnitGroup` defintitions and submit them as PRs if they're generally
 applicable.
 
+## Formatters
+
+Squants provides an experimental API for formatting Quantities in the "best unit." For example, 
```

Comment:
I guess either way the API will change but the experimental would at least be an indication.
I don't have a strong opinion either way